packages/react-server-test-pages/package.json

```diff
@@ -8,7 +8,6 @@
     "prepublish": "gulp build",
     "start": "react-server start",
     "test": "gulp test",
-    "debug": "node-debug --debug-brk=0 -p 9000 react-server start",
```

Comment:
I'm pretty sure we want to keep the debug target for test pages -- it's purpose is a place to repro and debug issues.